contracts/structs/JBFundAccessConstraints.sol

```diff
@@ -6,6 +6,8 @@ import './../interfaces/IJBPaymentTerminal.sol';
 struct JBFundAccessConstraints {
   // The terminal within which the distribution limit and the overflow allowance applies.
   IJBPaymentTerminal terminal;
+  // The token for which the distribution limit applies.
```

Comment:
Nit> "The token for which the fund access constraint applies."?

Reply:
nice